tests/integration/files/file/base/_modules/runtests_decorators.py

```diff
@@ -46,7 +46,7 @@ def missing_depends():
 
 
 @salt.utils.decorators.depends('time', fallback_function=_fallbackfunc)
-def depends_will_fallback():
+def depends_will_not_fallback():
```

Comment:
@cachedout how did you get this? I've just checked, only getting warning about 2 blank lines between functions.